schema/va-spec/base/insilico-tools.yml

```diff
@@ -0,0 +1,98 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/profiles/assay-var-effect-source.yaml"
+title: VA Spec Molecular Consequence statement result Profiles
+strict: true
+
+imports:
+  va-core: ./va-core-source.yaml
+  cat-vrs: ../../cat-vrs/cat-vrs-source.yaml
+  vrs: ../../vrs/vrs-source.yaml
+  gks-core: ../../gks-core/gks-core-source.yaml
+
+$defs:
+  InSilicoVariantEffectScoreStudyResult:
+      inherits: va-core:StudyResult
+      maturity: draft
+      type: object
+      description: >-
+        A StudyResult that reports a variant effect score from a functional assay.
+        Notes: Larry and Alex to discuss specifiedBy/sourceDataset/reportedIn to determine where everything will go
+      properties:
+        type:
+          extends: type
+          const: "InSilicoVariantEffectScoreStudyResult"
+          default: "InSilicoVariantEffectScoreStudyResult"
+          description: MUST be "InSilicoVariantEffectScoreStudyResult".
+        label:
+          type: string
+          description: A primary name for the score
+        focusVariant:
+          extends: focus
+          oneOf: 
+            - $refCurie: gks.core:iriReference
+            - $refCurie: vrs:Allele
+          description: The variant for which this annotation applies to.
+        transcripts:
+          type: array
+          ordered: false
+          items:
+            oneOf:
+              - $refCurie: gks.core:iriReference
+              - $refCurie: vrs:SequenceReference
+          description: Transcripts for which this score is applicable
+        score:
```

Comment:
'Transcripts for which this score is applicable' could be misleading - how is this defined?
For some methods, eg CADD, transcripts are used to create the prediction, but which specific transcript was used for which specific result is not simple to work out and these may be older than the transcripts used to predict the molecular effect.